samples/javascript_nodejs/51.teams-messaging-extensions-action/bots/teamsMessagingExtensionsActionBot.js

```diff
@@ -1,66 +1,78 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-const { TeamsActivityHandler, CardFactory, TeamsInfo } = require('botbuilder');
+const { TeamsActivityHandler, CardFactory, TeamsInfo, MessageFactory } = require('botbuilder');
+const configuration = require('dotenv').config();
```

Comment:
Check out the [usage example](https://www.npmjs.com/package/dotenv) - in particular, the first line in this file should be:

```javascript
require('dotenv').config();
```

Then, after the remaining `require`s, write:

```javascript
const env = process.env;
```